New package: FilePrompt

loquens opened the pull request:
Package allows opening and saving files without dialogs using Linux-shell-styled prompt with autocompletion and suggestions
<!--
Your pull request will be reviewed automatically and by a human.

The manual review may take several days or weeks,
depending on the reviewer's availability and workload.
If you haven't received a comment on your pull request
and it wasn't merged either,
it just hasn't been reviewed yet.

---

Please ensure the automated reviews pass.
Follow the instructions provided, if necessary.
You can speed up the process
by [running some tests locally](https://packagecontrol.io/docs/submitting_a_package#Step_7).

You can request a review from @packagecontrol-bot
to manually trigger an automated review
if you don't need to push a new commit.
Do **NOT** open a new pull request!

In general, make sure you:

 1. Used `"tags": true` and not `"branch": "master"`
    (versioning docs: <https://packagecontrol.io/docs/submitting_a_package#Step_4>)
 2. Added a README to your repository so that users (and reviewers) 
    can understand what your package provides.

You should proceed with a short description of what the package does
and, in case one or multiple similar package already exists,
why you believe it is different and needed
below this line. -->

braver commented:
Thanks for submitting your package. 

It looks like you're providing commands. It's recommended to always add them to the command palette via a Default.sublime-commands file ([example](https://github.com/SublimeLinter/SublimeLinter/blob/master/Default.sublime-commands)). You can also make it easier for users to add the suggested keybindings, by creating a [Main Menu entry](https://github.com/SublimeLinter/SublimeLinter/blob/master/Main.sublime-menu#L24).

How does this compare to existing packages like [FileDialog](https://packagecontrol.io/packages/FileDialog)?

loquens commented:
> Thanks for submitting your package.
> 
> It looks like you're providing commands. It's recommended to always add them to the command palette via a Default.sublime-commands file ([example](https://github.com/SublimeLinter/SublimeLinter/blob/master/Default.sublime-commands)). 
Fixed.

> You can also make it easier for users to add the suggested keybindings, by creating a [Main Menu entry](https://github.com/SublimeLinter/SublimeLinter/blob/master/Main.sublime-menu#L24).
Fixed.
Thank you for reviewing.

> How does this compare to existing packages like [FileDialog](https://packagecontrol.io/packages/FileDialog)?
FilePrompt is much simpler, provides only one thing (but customizable), has no dependencies and just works. FileDialog is unsupported now and goes in much bigger package [sublime-enchanced](https://github.com/shagabutdinov/sublime-enhanced). And it is not working (at least on my system) after installation from Package Control.
I've tried to find other packages providing similar functions, but found nothing.

braver commented on repository/f.json:
```diff
@@ -688,6 +688,18 @@
+			"name": "FilePrompt",
+			"details": "https://github.com/loquens/subl-open-file-prompt",
+			"labels": ["file navigation", "file manipulation", "utilities", "open files", "file utils", "file suggestions"],
+			"readme": "https://raw.githubusercontent.com/loquens/subl-open-file-prompt/main/README.md",
```
You don't need to provide this readme link. Everything is already found via the repo link in "details".

loquens replied:
Fixed. Seen it in other packages, so decided to add it.

braver commented:
Alrighty 🙆🏻‍♂️
